src/components/text_link.tsx

```diff
@@ -6,7 +6,8 @@ import "../assets/fonts"
 export interface LinkProps extends React.Props<TextLink>, React.HTMLAttributes<TextLink> {
   href?: string
   underline?: boolean
-  color?: string
+  color?: string,
+  style?: any,
```

Comment:
Regardless of the question why this uses inline styling, I’m surprised you’d need to specify this property and it should probably be of the [CSSProperties](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/299cbfa81a20929b0362815730239f9dcc992449/types/react/index.d.ts#L648) type.